src/main/java/team4/footwithme/stadium/service/StadiumServiceImpl.java

```diff
@@ -1,9 +1,81 @@
 package team4.footwithme.stadium.service;
 
 import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.locationtech.jts.geom.Point;
 import org.springframework.stereotype.Service;
+import team4.footwithme.global.util.PositionUtil;
+import team4.footwithme.stadium.exception.StadiumExceptionMessage;
+import team4.footwithme.stadium.service.response.StadiumDetailResponse;
+import team4.footwithme.stadium.service.response.StadiumsResponse;
+import team4.footwithme.stadium.domain.Stadium;
+import team4.footwithme.stadium.repository.StadiumRepository;
 
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Slf4j
 @RequiredArgsConstructor
 @Service
 public class StadiumServiceImpl implements StadiumService {
+
+    private final StadiumRepository stadiumRepository;
+
+    // 구장 목록 조회
+    public List<StadiumsResponse> getStadiumList() {
+        return stadiumRepository.findAll().stream()
+                .map(stadium -> new StadiumsResponse(stadium.getStadiumId(), stadium.getName(), stadium.getAddress()))
+                .toList();
+    }
+
+    // 구장 상세 정보 조회
+    public StadiumDetailResponse getStadiumDetail(Long id) {
+        Stadium stadium = findByIdOrThrowException(id);
+        Point position = stadium.getPosition();
+
+        return new StadiumDetailResponse(
+                stadium.getMember().getMemberId(),
+                stadium.getName(),
+                stadium.getAddress(),
+                stadium.getPhoneNumber(),
+                stadium.getDescription(),
+                position.getY(),
+                position.getX()
+        );
+    }
```

Comment:
저는 이렇게 길어지는게 싫어서 dto에서 of같은 메서드를 사용하곤합니다.
```suggestion
        return new StadiumDetailResponse(stadium,position);
    }
```
이러한 방식으로요 ! 한번 참고만 하셔도 될 거 같아요

Reply:
깔끔하네요! 참고하겠습니다